authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/services/UserService.kt

```diff
@@ -65,6 +66,18 @@ class UserService(
             .fetchMap(u.ORGANIZATION_ID, DSL.count())
     }
 
+    fun assertUserDoesNotExistInDbOrThrow(email: String) {
+        val u = Tables.USER
+
+        val user = dsl.selectFrom(u)
+            .where(u.EMAIL.eq(email))
+            .fetchOne()
+
+        if (user != null) {
+            conflict("User with this email already exists")
```

Comment:
Why providing the information with the same content as the exception? In this case the log would be 'User already exists. User with this email already exists'

Reply:
I don't quite get what you mean. This message gets passed to `WebApplicationException` which in turn gets passed along as an API response. I think this is the desired outcome here?

Reply:
Well, `conflict()` shouldn't repeat "user" which you also remarked and I did fix that